app/assets/stylesheets/player.scss

```diff
@@ -0,0 +1,60 @@
+.container-sign-up{
+  .div-btn {
+    text-align: center;
+  }
+
+  .btn-style {
+    width: 100%;
+    background-color: $secondary-font-color;
```

Comment:
Order you styles in alphabetical order. This makes it easier to locate the properties a lot faster and it's good practice.